### ISSUE TITLE
IdsCheckbox: A11Y: WCAG Guideline: 1.3.3 Sensory Characteristics and 1.4.11 Non-Text Contrast

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise-wc#1025]
**Describe the bug**
Issue 1. Information is conveyed in the form of symbol.
2. The color contrast ratio of checkboxes is less than the standard ratio 3:1.
3. The color contrast ratio of checkboxes is less than the standard ratio 3:1.

**To Reproduce**
Issue 1: 
WCAG Guideline: 1.3.3 Sensory Characteristics
Summary: Information is conveyed in the form of symbol.
Step to Reproduce: 
1. Launch the URL: https://main.wc.design.infor.com/ids-checkbox/example.html
2. Navigate to '*' symbol present on page and verify

Actual Result: Information is conveyed in the form of symbol. '*' symbol is used to declare a field as required. But it is not written anywhere that '*' symbol means required field.

Expect Results:Information should not be conveyed in the form of symbol. If it is conveyed then it should be written in textual form for eg. * means required field.

Instance Observation:
1. Checkbox> Side by Side
2. Checkbox> Standalone

User Impact: User will not be able to identify the purpose of that symbol.

Screenshot: https://www.screencast.com/t/0luasgky

Dev Recommendations: Information about that '*' symbol should be present in textual form. For eg. * means required field

Issue 2:
WCAG Guideline:1.4.11 Non-Text Contrast

Summary: The color contrast ratio of checkboxes is less than the standard ratio 3:1.

Step to Reproduce:
1. Launch the URL: https://main.wc.design.infor.com/ids-checkbox/example.html
2. Verify color contrast ratio of checkbox
 
Actual Result: The color contrast ratio of checkboxes having white foreground and green background is less than the standard ratio 3:1.

Expect Results:The color contrast ratio of checkboxes having white foreground and green background should be equal to or more than the standard ratio 3:1.

Instance Observation: 1. Checkbox> Standalone

User Impact: If Color contrast ratio fails for the text, then low vision user will not be able to access that particular text on the page.

Screenshot:https://www.screencast.com/t/gNWY6TR3Z3D

Dev Recommendations: Change the colour of the element with respect to background such that it is as per standard color contrast ratio i.e 3:1 (Foreground : Background)

Issue 3: 
WCAG Guideline: 1.4.11 Non-Text Contrast

Summary: The color contrast ratio of checkboxes is less than the standard ratio 3:1.

Step to Reproduce:
1. Launch the URL: https://main.wc.design.infor.com/ids-checkbox/standalone-css.html
2. Verify color contrast ratio of checkbox

Actual Result: The color contrast ratio of checkbox having white foreground and yellow background is less than the standard ratio 3:1.

Expect Results: The color contrast ratio of checkbox having white foreground and yellow background should be equal to or more than the standard ratio 3:1.

Instance Observation:

User Impact: If Color contrast ratio fails for the text, then low vision user will not be able to access that particular text on the page.

Screenshot: https://www.screencast.com/t/q1xazeByDRs

Dev Recommendations: Change the colour of the element with respect to background such that it is as per standard color contrast ratio i.e 3:1 (Foreground : Background)

**Platform**
 - Windows 10
 - Browser Name: Chrome

**Additional context**
Add any other context about the problem here. None

**Comment by danielortiz1982** [comment]
@inforandy Were going to need design insight on this color contrast ratio issue.
